hack/charts/training-jobs/templates/NOTES.txt

```diff
@@ -1,12 +1,8 @@
-Thanks for installing the {{ .Chart.Name }}. 
+Successfully configured a new training job.
```

Comment:
I wonder if this message will be confusing for new users. They might get the idea that the training job is correctly configured in SageMaker, which may or may not be true. Thoughts on removing?

Reply:
I did think about that scenario. Should I say "Successfully deployed a new training job chart"? It seems to be common to express some sort of "action succeeded" message upon installing the chart.

Reply:
I think that is a much better message.